cmd/version.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
+}
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print the version number of World Engine CLI",
+	Long:  `All software has versions. This is World Engine CLI's`,
+	Run: func(cmd *cobra.Command, args []string) {
+		// TODO: Make this read from a file
```

Comment:
we can read it from the running binary itself: https://tip.golang.org/doc/go1.18#go-version